### DISCUSSION TITLE
Numeric input instead of a slider

### DISCUSSION THREAD

**Discussion by pdwonline** [Clooos/Bubble-Card#741] [Feature requests]
I use some numeric input entities, for example for setting the room temperature or light offsets. Using bobble-card, I can use it as a slider, it would be nice to have the option to present it as a nummeric count up/down. See examples below:

<img width="242" alt="afbeelding" src="https://github.com/user-attachments/assets/c7c8d449-9b5c-4614-9def-02529fae9581">
(currently using the slider)

<img width="243" alt="afbeelding" src="https://github.com/user-attachments/assets/b18f0a75-f7bf-45df-85fa-2418b853ed82">
using a up/down counter. (functional the same as [Custom numberbox card](https://github.com/junkfix/numberbox-card)

**Comment by Clooos** [comment]
Hi, a climate card is in my plans but I have a pretty big to-do list 😅

**Reply by pdwonline** [reply]
No hurry, I can see you're very busy and appreciate your work!
